spec/controllers/api/v1/accounts/conversations/messages_controller_spec.rb

```diff
@@ -50,6 +50,7 @@
         expect(response_data['content_attributes']['external_created_at']).to eq time_stamp
         expect(conversation.messages.count).to eq(1)
         expect(conversation.messages.last.sender_id).to eq(agent_bot.id)
+        expect(conversation.messages.last.content_type).to eq('text')
```

Comment:
Can we add one more test with content blank and then check content type.

Reply:
^^ I guess we could just have the test in model spec since the code changes are done there.